Cargo.toml

```
@@ -69,6 +67,8 @@ leptos_actix = { version = "0.6.11" }
 thiserror = { version = "1.0.57" }
 leptos-use = "0.10.3"
 mime = "0.3.17"
+aws-sdk-kms = {version = "1.38.0"}
+aws-config = { version = "1.1.1", features = ["behavior-version-latest"] }
```

this is an un-supported version combination with respect to aws-sdk-kms, use:
```suggestion
aws-config = { version = "1.1.7", features = ["behavior-version-latest"] }
```

---

@ayushjain17 updated